testdata/csharp/csharp_8-dotnet_3/plan.json

```diff
@@ -2,5 +2,23 @@
   "dev_packages": [
     "dotnet-sdk_3"
   ],
-  "runtime_packages": []
+  "runtime_packages": [
+    "dotnet-sdk_3"
+  ],
+  "install_stage": {
+    "command": ""
+  },
+  "build_stage": {
+    "command": "dotnet publish",
+    "input_files": [
+      "."
+    ]
+  },
+  "start_stage": {
+    "command": "dotnet run",
+    "input_files": [
+      "."
+    ]
+  },
+  "definitions": null
```

Comment:
Not related to your change, but I'm not loving all these `null`s. I'd rather have an empty struct/array.